index.html

```diff
@@ -0,0 +1,568 @@
+<!doctype html>
```

Comment:
@davideshett what is going on here? why do we have a HTML file in this project?
Please delete it.